usecase/jobs.go

```diff
@@ -1,6 +1,8 @@
 package usecase
 
 import (
+	"math"
+	"sync"
 	"jobs/domain/model"
 	csvservice "jobs/service/csv"
 	httpservice "jobs/service/http"
```

Comment:
```suggestion
	"math"
	"sync"
	
	"jobs/domain/model"
	csvservice "jobs/service/csv"
	httpservice "jobs/service/http"
```
Your imported packages must be separated in three blocks (or 2 or 1, depending) by an empty line.

First block comsists on Golang's standard library packages.
Second block consists on your project packages.
Third block consists on 3rd party packages.

Also, take into consideration `jobs` is not a good name for your whole project. Normally you name it in your go.mod file as your repo URL; in your case, it would be something like: `github.com/jorgenino/jobs`, or something alike.